R/utils.R

```diff
@@ -72,6 +72,14 @@ write_over <- function(contents, path) {
   TRUE
 }
 
+append_to <- function(contents, path) {
+  stopifnot(is.character(contents), length(contents) == 1)
+
+  message("* Writing additional contents to '", path, "'")
```

Comment:
Can you please now use the `done()` and `value()` helpers?

This now also needs to go into `write.r` and probably be called `write_append()`. Can you please match the `base_path` and `path` arguments of the other functions so that you can print a nicer path in the message?